lib/calc.rb

```diff
@@ -1,3 +1,33 @@
 # frozen_string_literal: true
 
 # Calculator Class here
+
+class Calculator
+  def initialize(num1)
+    @num1 = num1
+  end
+
+  def add(num2)
+    @num1 += num2
+  end
+
+  def subtract(num2)
+    @num1 -= num2
+  end
+
+  def multtiply(num2)
+    @num1 *= num2
+  end
+
+  def divide(num2)
+    @num1 /= num2.to_f
+  end
+
+  def clear
+    @num1 = 0
+  end
+
+  def equals
+    p @num1
```

Comment:
You don't need to output the value to the console here, `@num1` by itself will work because of Ruby's implicit returns.